RecoBTag/ONNXRuntime/plugins/DeepDoubleXONNXJetTagsProducer.cc

```diff
@@ -128,6 +132,7 @@ void DeepDoubleXONNXJetTagsProducer::produce(edm::Event& iEvent, const edm::Even
     auto batch_size = std::count_if(
         tag_infos->begin(), tag_infos->end(), [](const auto& taginfo) { return !taginfo.features().empty(); });
 
+    std::vector<float> etas_; //DEBUG
```

Comment:
locals should not have a trailing underscore to avoid confusion with data members